bitcoin/src/sign_message.rs

```diff
@@ -115,7 +115,7 @@ mod message_signing {
             if bytes[0] < 27 {
                 return Err(secp256k1::Error::InvalidRecoveryId);
             };
-            let recid = RecoveryId::from_i32(((bytes[0] - 27) & 0x03) as i32)?;
+            let recid = RecoveryId::try_from(((bytes[0] - 27) & 0x03) as i32)?;
```

Comment:
Maybe this?

```rust
            let recid = RecoveryId::try_from(((bytes[0] - 27) & 0x03) as i32)
                .expect("cannot fail because of bitmask");
```